notebooks/6_calculate_statistics.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>This is an excellent attempt at explaining groupby with a diagram! However, I think it would be easier to understand if there were only 2 groups in the diagram, rather than 3. Part of the reason is that the 4 shades of green are difficult to distinguish between.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>